src/services/swap_service.ts

```diff
@@ -51,6 +52,7 @@ export class SwapService {
             rfqt: {
                 takerApiKeyWhitelist: RFQT_API_KEY_WHITELIST,
                 makerEndpoints: RFQT_MAKER_ENDPOINTS,
+                warningLogger: logger.warn.bind(logger),
```

Comment:
Is this bind meaningful? I don't think it is -- if this was only done because of a tslint issue, perhaps we can just `tslint:disable-next-line`.  If we need to keep the `.bind` I think it should be `.bind(this)` 🤔